prowler/providers/gcp/gcp_provider.py

```diff
@@ -56,7 +61,7 @@ def __init__(self, arguments):
         accessible_projects = self.get_projects()
         if not accessible_projects:
             logger.critical("No Project IDs can be accessed via Google Credentials.")
-            sys.exit(1)
+            raise SystemExit
```

Comment:
Add a TODO here to replace this with our custom exception.

Reply:
Done